packages/ember-runtime/tests/suites/array/includes.js

```diff
@@ -0,0 +1,35 @@
+import {SuiteModuleBuilder} from 'ember-runtime/tests/suites/suite';
+
+var suite = SuiteModuleBuilder.create();
+
+suite.module('includes');
```

Comment:
I'm unsure what you mean, maybe I am missing something. This code tests the implementation of `Array#includes`. How are the tests for `Enumerable#includes` applied to an array?